README.md

```diff
@@ -2,11 +2,39 @@
 
 A CLI tool for finding tickets that mention a team by no one has responded to from the team.
 
-`missed-issues --org mapbox --team teamname`
+`missed-issues --org mapbox --team teamname --token`
```

Comment:
`missed-issues --org mapbox --team teamname --token github_token` (just to follow the format you have)